tritium-jmh/build.gradle

```diff
@@ -24,6 +30,7 @@ dependencies {
     annotationProcessor 'org.openjdk.jmh:jmh-generator-annprocess'
 
     jmh project(':tritium-lib')
+    jmh project(':tritium-test')
     jmh 'ch.qos.logback:logback-classic'
     jmh 'com.palantir.remoting3:tracing'
     jmh 'org.slf4j:slf4j-simple'
```

Comment:
both slf4j-simple and logback-classpath on the classpath here, as well as slf4j-simple from tritium-test

Reply:
removed `slf4j-simple` from `tritium-test` and now we don't have the logger classpath conflict:


```
jmh
+--- project :tritium-lib
|    +--- project :tritium-api
|    |    \--- com.google.code.findbugs:jsr305 -> 3.0.2
|    +--- project :tritium-core
|    |    +--- project :tritium-api (*)
|    |    +--- com.google.guava:guava -> 23.6.1-jre
|    |    |    +--- com.google.code.findbugs:jsr305:1.3.9 -> 3.0.2
|    |    |    +--- org.checkerframework:checker-compat-qual:2.0.0
|    |    |    +--- com.google.errorprone:error_prone_annotations:2.1.3 -> 2.3.3
|    |    |    +--- com.google.j2objc:j2objc-annotations:1.1
|    |    |    \--- org.codehaus.mojo:animal-sniffer-annotations:1.14
|    |    +--- com.palantir.safe-logging:preconditions -> 1.8.2
|    |    |    +--- com.palantir.safe-logging:safe-logging:1.8.2
|    |    |    \--- com.google.errorprone:error_prone_annotations:2.3.2 -> 2.3.3
|    |    +--- com.palantir.safe-logging:safe-logging -> 1.8.2
|    |    \--- org.slf4j:slf4j-api -> 1.7.12
|    +--- project :tritium-metrics
|    |    +--- project :tritium-api (*)
|    |    +--- project :tritium-core (*)
|    |    +--- project :tritium-registry
|    |    |    +--- com.google.guava:guava -> 23.6.1-jre (*)
|    |    |    +--- com.palantir.safe-logging:preconditions -> 1.8.2 (*)
|    |    |    +--- com.palantir.safe-logging:safe-logging -> 1.8.2
|    |    |    \--- io.dropwizard.metrics:metrics-core -> 3.2.5
|    |    |         \--- org.slf4j:slf4j-api:1.7.22 -> 1.7.12
|    |    +--- io.dropwizard.metrics:metrics-core -> 3.2.5 (*)
|    |    +--- com.google.guava:guava -> 23.6.1-jre (*)
|    |    +--- com.palantir.safe-logging:preconditions -> 1.8.2 (*)
|    |    +--- com.palantir.safe-logging:safe-logging -> 1.8.2
|    |    +--- org.mpierce.metrics.reservoir:hdrhistogram-metrics-reservoir -> 1.1.2
|    |    |    \--- com.google.code.findbugs:jsr305:3.0.1 -> 3.0.2
|    |    +--- org.hdrhistogram:HdrHistogram -> 2.1.11
|    |    \--- org.slf4j:slf4j-api -> 1.7.12
|    +--- project :tritium-proxy
|    +--- project :tritium-slf4j
|    |    +--- project :tritium-api (*)
|    |    +--- project :tritium-core (*)
|    |    +--- com.google.guava:guava -> 23.6.1-jre (*)
|    |    +--- com.palantir.safe-logging:preconditions -> 1.8.2 (*)
|    |    +--- com.palantir.safe-logging:safe-logging -> 1.8.2
|    |    \--- org.slf4j:slf4j-api -> 1.7.12
|    +--- project :tritium-tracing
|    |    +--- project :tritium-api (*)
|    |    +--- project :tritium-core (*)
|    |    +--- com.palantir.safe-logging:preconditions -> 1.8.2 (*)
|    |    +--- com.palantir.safe-logging:safe-logging -> 1.8.2
|    |    +--- com.palantir.tracing:tracing -> 2.9.0
|    |    |    +--- com.palantir.tracing:tracing-api:2.9.0
|    |    |    |    +--- com.fasterxml.jackson.core:jackson-databind:2.9.8
|    |    |    |    |    +--- com.fasterxml.jackson.core:jackson-annotations:2.9.0 -> 2.9.8
|    |    |    |    |    \--- com.fasterxml.jackson.core:jackson-core:2.9.8
|    |    |    |    \--- com.fasterxml.jackson.core:jackson-annotations:2.9.8
|    |    |    +--- com.fasterxml.jackson.core:jackson-databind:2.9.8 (*)
|    |    |    +--- com.fasterxml.jackson.datatype:jackson-datatype-guava:2.9.8
|    |    |    |    +--- com.google.guava:guava:18.0 -> 23.6.1-jre (*)
|    |    |    |    +--- com.fasterxml.jackson.core:jackson-core:2.9.8
|    |    |    |    \--- com.fasterxml.jackson.core:jackson-databind:2.9.8 (*)
|    |    |    +--- com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.9.8
|    |    |    |    +--- com.fasterxml.jackson.core:jackson-core:2.9.8
|    |    |    |    \--- com.fasterxml.jackson.core:jackson-databind:2.9.8 (*)
|    |    |    +--- com.fasterxml.jackson.module:jackson-module-afterburner:2.9.8
|    |    |    |    +--- com.fasterxml.jackson.core:jackson-core:2.9.8
|    |    |    |    \--- com.fasterxml.jackson.core:jackson-databind:2.9.8 (*)
|    |    |    +--- com.google.guava:guava:27.0.1-jre -> 23.6.1-jre (*)
|    |    |    +--- com.palantir.safe-logging:preconditions:1.8.2 (*)
|    |    |    +--- com.palantir.safe-logging:safe-logging:1.8.2
|    |    |    +--- org.slf4j:slf4j-api:1.7.12
|    |    |    +--- com.fasterxml.jackson.core:jackson-annotations:2.9.8
|    |    |    +--- com.fasterxml.jackson.core:jackson-core:2.9.8
|    |    |    +--- com.google.code.findbugs:jsr305:3.0.2
|    |    |    \--- com.google.errorprone:error_prone_annotations:2.3.2 -> 2.3.3
|    |    \--- org.slf4j:slf4j-api -> 1.7.12
|    +--- com.google.guava:guava -> 23.6.1-jre (*)
|    +--- com.palantir.safe-logging:preconditions -> 1.8.2 (*)
|    +--- com.palantir.safe-logging:safe-logging -> 1.8.2
|    \--- org.slf4j:slf4j-api -> 1.7.12
+--- project :tritium-test
|    +--- org.slf4j:slf4j-api -> 1.7.12
|    +--- project :tritium-api (*)
|    +--- com.google.guava:guava -> 23.6.1-jre (*)
|    +--- com.palantir.safe-logging:preconditions -> 1.8.2 (*)
|    +--- com.palantir.safe-logging:safe-logging -> 1.8.2
|    \--- io.dropwizard.metrics:metrics-core -> 3.2.5 (*)
+--- ch.qos.logback:logback-classic -> 1.1.2
|    +--- ch.qos.logback:logback-core:1.1.2
|    \--- org.slf4j:slf4j-api:1.7.6 -> 1.7.12
+--- com.palantir.remoting3:tracing -> 3.45.0
|    +--- com.palantir.remoting3:jackson-support:3.45.0
|    |    +--- com.fasterxml.jackson.core:jackson-databind:2.9.5 -> 2.9.8 (*)
|    |    +--- com.fasterxml.jackson.datatype:jackson-datatype-guava:2.9.5 -> 2.9.8 (*)
|    |    +--- com.fasterxml.jackson.module:jackson-module-afterburner:2.9.5 -> 2.9.8 (*)
|    |    +--- com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.9.5 -> 2.9.8 (*)
|    |    +--- com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.9.5 -> 2.9.8
|    |    |    +--- com.fasterxml.jackson.core:jackson-annotations:2.9.0 -> 2.9.8
|    |    |    +--- com.fasterxml.jackson.core:jackson-core:2.9.8
|    |    |    \--- com.fasterxml.jackson.core:jackson-databind:2.9.8 (*)
|    |    \--- com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:2.9.5 -> 2.9.8
|    |         \--- com.fasterxml.jackson.core:jackson-core:2.9.8
|    +--- com.fasterxml.jackson.core:jackson-databind:2.9.5 -> 2.9.8 (*)
|    +--- com.fasterxml.jackson.datatype:jackson-datatype-guava:2.9.5 -> 2.9.8 (*)
|    +--- com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.9.5 -> 2.9.8 (*)
|    +--- com.fasterxml.jackson.module:jackson-module-afterburner:2.9.5 -> 2.9.8 (*)
|    +--- com.google.guava:guava:20.0 -> 23.6.1-jre (*)
|    +--- com.palantir.tracing:tracing:1.2.0 -> 2.9.0 (*)
|    +--- com.palantir.remoting-api:tracing:1.9.0
|    |    \--- com.fasterxml.jackson.core:jackson-databind:2.6.7 -> 2.9.8 (*)
|    +--- com.palantir.safe-logging:safe-logging:1.3.0 -> 1.8.2
|    \--- org.slf4j:slf4j-api:1.7.12
+--- org.slf4j:slf4j-simple -> 1.7.12
|    \--- org.slf4j:slf4j-api:1.7.12
+--- org.openjdk.jmh:jmh-core:1.21
|    +--- net.sf.jopt-simple:jopt-simple:4.6
|    \--- org.apache.commons:commons-math3:3.2
\--- org.openjdk.jmh:jmh-generator-bytecode:1.21
     +--- org.openjdk.jmh:jmh-core:1.21 (*)
     +--- org.openjdk.jmh:jmh-generator-reflection:1.21
     |    \--- org.openjdk.jmh:jmh-core:1.21 (*)
     \--- org.openjdk.jmh:jmh-generator-asm:1.21
          +--- org.openjdk.jmh:jmh-core:1.21 (*)
          +--- org.openjdk.jmh:jmh-generator-reflection:1.21 (*)
          \--- org.ow2.asm:asm:5.0.3
```